pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1454,7 +1486,7 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
 
         // If it's not a privileged installation preset guestAgent
         // port binding address to localhost only.
-        if (!this.privilegedServiceEnabled && !config.kubernetes.ingress.localhostOnly) {
+        if (!this.getIsAdminInstall() && !config.kubernetes.ingress.localhostOnly) {
```

Comment:
This doesn't match the comment. I'm having issues figuring out which of the two is correct; I think `privilegedServiceEnabled` might be the more correct one?

Also, if we _do_ want to use `getIsAdminInstall()` here, this needs an `await`.

Reply:
Updated, we were piggybacking off of `privilegedServiceEnabled` to determine admin Installation, however, that is not going to work in the New Network.